target/linux/pistachio/creator-platform-default.config

```diff
@@ -3,20 +3,9 @@ CONFIG_TARGET_pistachio_marduk=y
 CONFIG_TARGET_pistachio_marduk_marduk_cc2520=y
 CONFIG_IMAGEOPT=y
 CONFIG_LIBCURL_OPENSSL=y
```

Comment:
Yes it does. since we are doing a cat target/linux/pistachio/creator-platform-default-cascoda.config > .config followed by make defconfig, it gets reflected properly.